src/Limbo.Integrations.Skyfish/Http/SkyfishHttpClient.cs

```diff
@@ -113,6 +117,12 @@ public class SkyfishHttpClient : HttpClient {
             return JsonUtils.ParseJsonObject(response.Body, SkyfishVideo.Parse);
         }
 
+        private string GetVideoThumbnail(int videoId) {
+            var response = Get($"/search?unique_media_id={videoId}&return_values=unique_media_id+height+width+title+description+thumbnail_url+thumbnail_url_ssl+filename+file_disksize+file_mimetype");
+            SkyfishVideo video = JsonUtils.ParseJsonObject(response.Body, SkyfishVideo.Parse);
+            return video.ThumbnailUrlSsl;
+        }
+
```

Comment:
Why the private method? Couldn't the method body just be part of the `GetThumbnailUrl`?